tests/test_cache.py

```diff
@@ -0,0 +1,16 @@
+from unittest import TestCase
+from canonicalwebteam.http import create_cached_session
+from datetime import timedelta
+
+
+class TestCachedSession(TestCase):
+    def test_custom_heuristic(self):
+        session = create_cached_session(fallback_cache_duration=10)
+        adapter = session.get_adapter("http://")
+        heuristic = adapter.heuristic
+        delta = heuristic.delta
+        self.assertEqual(delta, timedelta(seconds=10))
+
+
+if __name__ == "__main__":
+    unittest.main()
```

Comment:
If you want to have this you should `import unittest`.

I try to run usually tests with `python -m unittest discover tests` which doens't need this block.
So you can remove those two lines :smile:

Reply:
I would suggest leaving those two lines. Providing people with the option to run your python tests directly is simply good manners.

But yes, if you want to use `unittest`, you need to `import` it.